tests/openthread_sock_udp/Makefile

```diff
@@ -0,0 +1,52 @@
+APPLICATION = openthread_sock_udp
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= samr21-xpro
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+
+CFLAGS += -DDEVELHELP -Wall
+
+#Uncomment the following line for running OpenThread CLI
+#CFLAGS += -DMODULE_OPENTHREAD_CLI
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+USEMODULE += openthread
+USEMODULE += openthread_contrib
+USEMODULE += openthread_sock_udp
+USEMODULE += openthread_sock
+USEMODULE += core_mbox
+
+
+ifneq (,$(filter samr21-xpro,$(BOARD)))
```

Comment:
I think that, given the board type, the right driver module is automatically added. So do we really need those lines ?